src/site/xdoc/mappers.xml

```diff
@@ -247,7 +247,11 @@ public class AppConfig {
 	  You can also provide an specific <code>SqlSessionFactory</code> or <code>SqlSessionTemplate</code>
 	  by using its properties <code>sqlSessionFactory</code> and <code>sqlSessionTemplate</code>.
 	  </p>
-     
+
+    <p>
+      <span class="label important">NOTE</span> Since 2.0.4, If <code>basePackageClasses</code> or <code>basePackages</code> are not defined, scanning will occur from the package of the class that declares this annotation.
```

Comment:
Nice working!! Could you copy same content to other languages(es, ja, ko, zh)?

Reply:
Added!

Reply:
thx!